tests/rptest/services/redpanda.py

```diff
@@ -966,6 +974,32 @@ def write_tls_certs(self):
                 node.account.copy_to(self._pandaproxy.pp_client_crt,
                                      ProxyConfig.PP_TLS_CLIENT_CRT_FILE)
 
+            if self._schema_registry.sr_client_key is not None:
```

Comment:
I would break out the changes in this commit that modify redpanda.py from the other changes that write actual new tests